filament/backend/test/test_ReadPixels.cpp

```diff
@@ -16,15 +16,38 @@
 
 #include "BackendTest.h"
 
+#include <imageio/ImageEncoder.h>
+
+#include <image/ColorTransform.h>
+
 #include "ShaderGenerator.h"
 #include "TrianglePrimitive.h"
 
 #include <utils/Hash.h>
 
 #include <fstream>
 
+using namespace filament;
+using namespace filament::backend;
+using namespace image;
+
 namespace {
 
+template<typename T>
+static LinearImage toLinear(size_t w, size_t h, size_t bpr, const uint8_t* src) {
```

Comment:
Don't we have multiple copies of this method over the code base now? Worth having only one?

Reply:
Good catch, added a commit.